### PR TITLE
[WIP] Rtk record

### PR DESCRIPTION
**Pull request by ethanrublee**
- Hook up the tcp stream to frontend.
- TODO record the track of RTK and display on webapp.

### CODE REVIEW

File: python/farm_ng/tractor/frontend/static/rtkrover_solution.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	// field_names = ['date', 'time_gps_utc', 'time_rcv_utc', 'e_baseline_m', 'n_baseline_m',
+	//                'u_baseline_m', 'status', 'n_satelites', 'std_e_m', 'std_n_m',
+	//                'std_u_m', 'sde_en_m', 'sde_nu_m', 'sde_ue_m', 'age', 'ar_ratio',
+	//                'velocity_e_ms', 'velocity_n_ms', 'velocity_u_ms', 'std_ve',
+	//                'std_vn', 'std_vu', 'std_ven', 'std_vnu', 'std_vue']
+
+	// 'fix','float', 'single' - unlikely to be 'sbas', 'dgps', 'ppp'
+        $("#rtk_status").text(message.status);
+
+	$("#rtk_baseline").text('e:' + message.e_baseline_m + ' n:' + message.n_baseline_m + ' u:'+ message.u_baseline_m);
```
Comment (ethanrublee) [inline python/farm_ng/tractor/frontend/static/rtkrover_solution.js:44]: @mabitter heads up, adding support for parsed realtime data stream from the rtk rover.

**Comment by ethanrublee** [comment]
https://youtu.be/0xlpqyHxWiM

Screenshot from 2020-04-27 23-46-38

**Comment by mabitter** [comment]
@ethanrublee - Sick.  
It looks great.  I'm like Siskel and Ebert if you ever want a two thumbs up on your code reviews, just ping me directly otherwise i'll assume you want one of the guys to take a look.

**Comment by ethanrublee** [comment]
[tractor_sim_demo](http://www.youtube.com/watch?v=H9RRonGdkLI)

**Pull request merged**